OSX/Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<string>1</string>
 	<key>NSHumanReadableCopyright</key>
 	<string>Copyright © 2016 Bugsnag. All rights reserved.</string>
```

Comment:
nit: while I'm looking at this, the copyrights could be updated.

Reply:
Updated copyright